src/controllers/authenticationController.ts

```diff
@@ -224,11 +234,20 @@ export default class AuthenticationController {
       this.lollipopParams.isLollipopEnabled &&
       O.isSome(errorOrMaybeAssertionRef.right)
     ) {
+      const assertionRefToRevoke = errorOrMaybeAssertionRef.right.value;
       // Sending a revoke message for previous assertionRef related to the same fiscalCode
       // This operation is fire and forget
       this.lollipopParams.lollipopService
-        .revokePreviousAssertionRef(errorOrMaybeAssertionRef.right.value)
+        .revokePreviousAssertionRef(assertionRefToRevoke)
         .catch(err => {
+          this.appInsightsTelemetryClient?.trackEvent({
```

Comment:
Add test

Reply:
done in https://github.com/pagopa/io-backend/pull/996/commits/06eceba5db872dc0e45f3d08e6925e71a721e63e